metron-interface/metron-alerts/src/app/pcap/model/pcap.request.ts

```diff
@@ -17,13 +17,13 @@
  */
 
 export class PcapRequest {
-  startTimeMs: number = 0;
-  endTimeMs: number = 150000000000000000;
-  ipSrcAddr: string = '';
-  ipSrcPort: number;
-  ipDstAddr: string = '';
-  ipDstPort: number;
-  protocol: string = '';
-  packetFilter: string = '';
-  includeReverse: boolean = false;
+  startTimeMs = 0;
+  endTimeMs = 150000000000000000;
```

Comment:
Why the magic number?

We use this value to produce a formatted date string by using the Date constructor. But

```js
new Date(150000000000000000) // -> Invalid Date
```

so we have to do extra checks to avoid showing "Invalid Date" in the date range selector:
https://github.com/tiborm/metron/blob/METRON-1724/metron-interface/metron-alerts/src/app/pcap/pcap-filters/pcap-filters.component.ts#L59

which makes it hard to understand and maintain the code.

Using magic numbers is a bad practice because it's hard to understand their meaning. We should add a hint as a code comment at least. Or use another, more straightforward value. 
My first thought was `Date.now()` but it would introduce side effects so I let it go.

What do you think?

Reply:
I agree. These default values should be fixed. Do we needs to assign value to the dates here?

Reply:
I think it would be better to leave them undefined here and let this file to be a simple type definition.

Reply:
It's not necessary to set these values here but I'm afraid the server relies on it as default values. So it might crash if we send it undefined values instead of magic numbers.